src/services/auth.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+
+const SignUpUser = (name,email,password)=>{
+    return axios.post('/api/auth/signup',{
```

Comment:
you shoiuld add error handling here

Reply:
> you shoiuld add error handling here

I have done it in context only and I don't think there will be any problem but if required I refactor it later.